cli/azd/pkg/project/framework_service_dotnet.go

```diff
@@ -42,16 +42,46 @@ func (dp *dotnetProject) Package(ctx context.Context, progress chan<- string) (s
 }
 
 func (dp *dotnetProject) InstallDependencies(ctx context.Context) error {
-	dotnetCli := tools.NewDotNetCli()
-	if err := dotnetCli.Restore(ctx, dp.config.Path()); err != nil {
+	if err := dp.dotnetCli.Restore(ctx, dp.config.Path()); err != nil {
 		return err
 	}
 	return nil
 }
 
+func (dp *dotnetProject) Initialize(ctx context.Context) error {
+	if err := dp.dotnetCli.InitializeSecret(ctx, dp.config.Path()); err != nil {
+		return err
+	}
+
+	handler := func(ctx context.Context, args ServiceLifecycleEventArgs) error {
+		bicepOutputArgs := args.Args["bicepOutput"]
+		if bicepOutputArgs == nil {
+			return fmt.Errorf("fail on interface conversion: interface {} is nil")
```

Comment:
nit: The error message is incorrect here.  `args` just doesn't contain the `bicepOutput`.  So handler has nothing to do and can just exit?